### DISCUSSION TITLE
Static report generation

### DISCUSSION THREAD

**Discussion by vkorn** [michenriksen/gokiburi#4] [Ideas]
Following up on reddit discussion. 

Not sure if anything else uses coverage report this way, but that's how we setup our build process:
1. Run linter/tests
2. Package/dockerize everything
3. Cleanup the coverage (usual `cat cover/cover.tmp.out | grep -v ".pb.go" >> cover/cover.out`)
4. Generate html report and publish it as a build artifact, later available through the deployment pipeline

Default go cover html looks, well, bad and doesn't have information about tests themselves. The view `gokiburi` is way more informative. It would be amazing if we can run `gokiburi` instead of test/coverage and have a single html file as an output. With packages/tests/coverage details.

**Comment by michenriksen** [comment]
Thanks for writing it up in here! I think it's a really cool idea. It sticks a bit outside what I see the main Gokiburi tool being for now, but I can definitely see this being a small, CI-friendly auxiliary tool for Gokiburi that I will look into.
